web/client/epics/widgets.js

```diff
@@ -199,7 +199,7 @@ export const toggleWidgetConnectFlow = (action$, {getState = () => {}} = {}) =>
                             if (widget.widgetType === 'map') {
                                 deps = deps.filter(d => (WIDGETS_MAPS_REGEX.exec(d) || [])[2] === widget.selectedMapId);
                             }
-                            return configureDependency(active, deps[0], options, widget.dependeciesMap).concat(Rx.Observable.of(toggleDependencySelector(false, {})));
+                            return configureDependency(active, deps[0], options, widget.dependenciesMap).concat(Rx.Observable.of(toggleDependencySelector(false, {})));
```

Comment:
I think this is a fix for another functionality. @dsuren1 is working on widget's fixes too, tell to him this and remove from this PR.

Reply:
correct

Reply:
The argument is not used by the function. Removed it in the widget fix PR

Reply:
So please remove this fix from your PR too @MV88 , so we can merge your contributions easely.